test/Microsoft.ML.Tests/Scenarios/Api/ApiScenariosTests.cs

```diff
@@ -43,14 +44,14 @@ public class IrisPrediction
         public class SentimentData
         {
             [ColumnName("Label")]
-            public bool Sentiment;
+            public DvBool Sentiment;
```

Comment:
Same here - shouldn't these stay as `bool`?

Reply:
Not sure how this happened. Reverted the file.

---
In reply to: [212432157](https://github.com/dotnet/machinelearning/pull/683#discussion_r212432157) [](ancestors = 212432157)